src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -1,9 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <PropertyGroup>
```

Comment:
Just asking this for my own understanding. Why is this one not required anymore? Is this because its picking the PackageVersion from src/System.Data.SqlClient/dir.props which is already 4.6.0?

Reply:
That's correct. Sometimes the version get added directly the pkgproj and we missed updating/removing that. Ideally you would have caught this with your local build and noticed the package produced didn't have the version you expected.

Reply:
Got it! Thanks.